### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by emirotin**
The main change is updating to etcher-image-writer@^9.

This requires updating the fake drive file to match the size of the image being written

### CODE REVIEW

**Comment by emirotin** [comment]
FYI the tests pass but I'm going to manually test it in integration with the CLI.

File: lib/commands.coffee [hunk 1]
```diff
@@ -60,21 +61,25 @@ module.exports =
...
-			imageReadStream = fs.createReadStream(operation.image)
-
-			# This is read by Resin Image Write to
-			# emit correct `progress` events.
-			imageReadStream.length ?= size
-
-			return imageWrite.write(options.drive, imageReadStream)
+			Promise.props
+				imageSize: fs.statAsync(image).get('size')
+				imageStream: Promise.resolve(fs.createReadStream(image))
```
Comment (pimterry) [inline lib/commands.coffee]: Nit: You don't need a `Promise.resolve` here, Bluebird will happily treat bare values as if they'd come from promises.

Comment (emirotin) [inline lib/commands.coffee]: Yeah, copied it from a different place.

**Comment by emirotin** [comment]
Huh, naive drive size detection doesn't work for the real devices (as opposed to files used in the tests), will have to make use of `drivelist`

**Pull request merged**